cmd/runtimetest/main.go

```diff
@@ -588,6 +588,8 @@ func validate(context *cli.Context) error {
 		return err
 	}
 
+	platform := context.String("platform")
```

Comment:
We also need to pass this to `validation_test.go#getDefaultGenerator` and change the Makefile (by adding a PLATFORM for example).